src/Main.java

```diff
@@ -1,5 +1,86 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("Задача 1");
+        int plannedCapital = 2459000;
+        int monthlyAccumulation = 15000;
+        int totalAccumulation = 0;
+        int months = 0;
+
+        while (totalAccumulation < plannedCapital) {
+            totalAccumulation += monthlyAccumulation;
+            months++;
+            System.out.println("Месяц " + months + ", сумма накоплений равна " + totalAccumulation + " рублей");
+        }
+
+        System.out.println("Задача 2");
+        int start = 1;
+        while (start <= 10) {
+            System.out.print(start + " ");
+            start = start + 1;
+        }
+        System.out.println();
+            for (int start2 = 10; start2 >= 1; start2--) {
+            System.out.print(start2 + " ");
+        }
+        System.out.println();
+        System.out.println("Задача 3");
+        int y = 12_000_000;
+        int birth = 17;
+        int mortality = 8;
+        int year = 0;
+        int changesOn = 1000;
+        while (year <= 10) {
+            year++;
+            y += ((y / changesOn) * (birth - mortality));
+            System.out.println("Год " + year + " , численность населения составляет " + y);
+        }
+        System.out.println("Задача 4");
+        int deposit = 15000;
+        double percentMonth = 0.07;
+        int accumulation = 12_000_000;
+        int month = 1;
+        while (deposit <= accumulation) {
+            deposit += deposit * percentMonth;
+            System.out.println("В " + month + " месяц, сумма накоплений составит: " + deposit);
+            month++;
+        }
+        System.out.println("Задача 5");
+        int deposit2 = 15000;
+        double percentMonth2 = 0.07;
+        int accumulation2 = 12_000_000;
+        int month2 = 0;
+        while (deposit2 <= accumulation2) {
+            deposit2 = (int) (deposit2 * (1 + percentMonth2));
+            month2++;
+            if (month2 % 6 == 0) {
+                System.out.println("В " + month2 + " месяц, сумма накоплений составит: " + deposit2);
+            }
+        }
+        System.out.println("Задача 6");
+        months = 0;
+        int deposit3 = 15000;
+        double percent = 0.07;
+        int monthsMax = 108;
```

Comment:
откуда 108? в уме не нужно никогда ничего считать. Было дано 9 лет, значит так и надо записать в переменную
```suggestion
        int monthsMax = 9 * 12;
```